Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -56,6 +57,14 @@ struct CommitTreeState {
     HashSet<ScrollingNodeID> synchronousScrollingNodes;
     // orphanNodes keeps child nodes alive while we rebuild child lists.
     OrphanScrollingNodeMap orphanNodes;
+    // Hosted subtrees needing attaching to scrolling tree after main commit has finished
+    Vector<std::pair<LayerHostingContextIdentifier, Vector<std::unique_ptr<ScrollingStateTree>>>> pendingSubtreesNeedingCommit;
+    // Nodes that are descendants of a frame hosting node.
+    HashSet<ScrollingNodeID> hostedScrollingNodes;
```

Comment:
The name should communicate that these are all subframe nodes.

Reply:
So these are just the list of all descendants of the frameHostingNode not just subframe nodes.